app/Http/Controllers/Api/v1/StudentApiController.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Http\Controllers\Api\v1;
+
+use App\Models\Student;
+use Illuminate\Http\Request;
+use App\Http\Controllers\Controller;
+
+class StudentApiController extends Controller
+{
+    public function search (Request $request)
+    {
+        $students = Student::with(['classroom', 'level'])
```

Comment:
We need to introduce an indexed search term field.

Reply:
that's sounds like a gud idea